From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andras Becsi <andras.becsi@digia.com>
Date: Tue, 6 May 2014 17:05:55 +0200
Subject: Do not use icu-config on eLinux

We require that the icu libs are in the default library seach path.
---
 icu.gyp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/icu.gyp b/icu.gyp
index 4d8259a..416050d 100644
--- a/icu.gyp
+++ b/icu.gyp
@@ -642,6 +642,8 @@
                 ],
               },
             },{ # OS!="android"
+              'conditions': [
+              ['qt_os!="embedded_linux"', {
               'link_settings': {
                 'ldflags': [
                   '<!@(icu-config --ldflags)',
@@ -650,6 +652,8 @@
                   '<!@(icu-config --ldflags-libsonly)',
                 ],
               },
+              }],
+              ],
             }],
           ],
         },
